main.js

```diff
@@ -0,0 +1,27 @@
+const http = require('http');
+const router = require('./src/router');
+const { port } = require('./config');
+
+const startServer = port => {
+
+  const server = http.createServer((req, res) => {
+    try {
+      !router.has(req.url) && res.writeHead(404).end('Not found');
```

Comment:
Better to write it in `if` for readability. And make return. Since second `.end` call would cause error.

Reply:
done